doc/build.md

```diff
@@ -54,6 +56,18 @@ $ cmake -DCMAKE_BUILD_TYPE=RelWithDebInfo -DCMAKE_PREFIX_PATH="$(pwd)/../third-p
 $ cmake --build . -- -j $(nproc)
 ```
 
+### Mac build commands
+Mac environments default to a proprietary version of LLVM and Clang which do not work with ML-IO build. It is recommended to use the LLVM that is installed via Homebrew; this version is usually found in ```/usr/local/opt/llvm/bin/clang``` after installation.
+
+```bash
+$ git clone https://github.com/awslabs/ml-io.git mlio
+$ cd mlio
+$ build-tools/build-dependency build/third-party all
+$ mkdir build/release
+$ cd build/release
+$ cmake -DCMAKE_C_COMPILER=/usr/local/opt/llvm/bin/clang -DCMAKE_BUILD_TYPE=RelWithDebInfo -DCMAKE_PREFIX_PATH="$(pwd)/../third-party" ../..
+$ cmake --build . -- -j $(sysctl -n hw.physicalcpu)
```

Comment:
Why not using: sysctl -n hw.ncpu

Reply:
You are right, nproc is processing units not physical CPU. Will change.